src/nvim/option.c

```diff
@@ -2764,14 +2756,6 @@ bool get_tty_option(const char *name, char **value)
     return true;
   }
 
-  if (is_tty_option(name)) {
-    if (value) {
-      // XXX: All other t_* options were removed in 3baba1e7.
-      *value = xstrdup("");
```

Comment:
This was here to allow old vimrcs to be used without complaint. Perhaps now it's time to show an error if they're set (there might be a common area where we do that, though we'll likely still need `is_tty_option`).